setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+#Script to run To-do app using the bash script
+
+#Step1: Download package from the repository  
+sudo apt-get update -y
+
+#Step2: Install PyEnv Requisites using apt get
+sudo apt-get install -y git
+
+#Step3: Installs pip Python3 Version 
+sudo apt-get install python3-pip
+
+#Step4: Copy poetry installation file to local 
+curl -sSL https://raw.githubusercontent.com/python-poetry/ \
```

Comment:
Worth noting that not all machines will come with curl by default, also installing curl as you did git would be worthwhile